transform.Rmd

```diff
@@ -155,743 +154,655 @@ NA + 10
+Hay tres pasos para preparar esta información:
 
-1.  Group flights by destination.
+1. Agrupa los vuelos por destino.
 
-1.  Summarise to compute distance, average delay, and number of flights.
+2. Resume en cada grupo para calcular la distancia, la demora promedio y el número de vuelos.
 
-1.  Filter to remove noisy points and Honolulu airport, which is almost
-    twice as far away as the next closest airport.
+3. Filtra para eliminar puntos ruidosos y el aeropuerto de Honolulu, que está casi dos veces más lejos que el próximo aeropuerto más cercano.
 
-This code is a little frustrating to write because we have to give each intermediate data frame a name, even though we don't care about it. Naming things is hard, so this slows down our analysis. 
+Nombrar cosas es difícil y enlentece nuestro análisis. Es entonces un poco frustrante escribir este código porque tenemos que dar un nombre a cada *data frame* intermedio, incluso si el *data frame* en sí mismo no nos importa.
 
-There's another way to tackle the same problem with the pipe, `%>%`:
+Hay otra forma de abordar el mismo problema con un *pipe*, %>%:
 
 ```{r}
-delays <- flights %>% 
-  group_by(dest) %>% 
+delays <- vuelos %>% 
+  group_by(destino) %>% 
   summarise(
     count = n(),
-    dist = mean(distance, na.rm = TRUE),
-    delay = mean(arr_delay, na.rm = TRUE)
+    distancia = mean(distancia, na.rm = TRUE),
+    atraso = mean(atraso_llegada, na.rm = TRUE)
   ) %>% 
-  filter(count > 20, dest != "HNL")
+  filter(count > 20, destino != "HNL")
 ```
 
-This focuses on the transformations, not what's being transformed, which makes the code easier to read. You can read it as a series of imperative statements: group, then summarise, then filter. As suggested by this reading, a good way to pronounce `%>%` when reading code is "then".
+Esto se enfoca en las transformaciones, no en lo que se está transformando, lo que hace que el código sea más fácil de leer. Puedes leerlo como una serie de declaraciones imperativas: agrupa, luego resume y por último filtra. Como sugiere esta lectura, una buena forma de pronunciar `%>%` cuando se lee el código es "y luego".  
 
-Behind the scenes, `x %>% f(y)` turns into `f(x, y)`, and `x %>% f(y) %>% g(z)` turns into `g(f(x, y), z)` and so on. You can use the pipe to rewrite multiple operations in a way that you can read left-to-right, top-to-bottom. We'll use piping frequently from now on because it considerably improves the readability of code, and we'll come back to it in more detail in [pipes].
+Lo que encuentras detrás del código, es que `x %>% f(y)` se convierte en `f(x, y)`, y `x %>% f(y) %>% g(z)` se convierte en `g(f(x, y), z)` y así sucesivamente. Puedes usar un *pipe* para reescribir múltiples operaciones de forma que puedas leer de izquierda a derecha, de arriba hacia abajo. Usaremos *pipes* con frecuencia a partir de ahora porque mejora considerablemente la legibilidad del código, y volveremos al tema con más detalle en [pipes].
 
```

Comment:
- Creo que sería mejor traducir _behind the scenes_ directamente como _detrás del código_. La oración quedaría: Detrás del código, `x %>% f(y)` se convierte en `f(x, y)` ...
- Puedes usar el *pipe* para ....